examples/remarketing/upload_offline_conversion.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example imports offline conversion values for specific clicks.
+
+To get Google Click ID for a click, use the "click_view" resource:
+https://developers.google.com/google-ads/api/fields/latest/click_view.
+To set up a conversion action, run the AddConversionAction.php example.
```

Comment:
You probably intend to reference the Python example, rather than the PHP one here.

Reply:
fixed